config/default.yaml

```
@@ -289,15 +289,15 @@ webhooks:
   maxBytes: 1048576 # 1MB
 
 coverage:
-  default: true
-  # plugin: sonar
-  # sonar:
-  #   sdApiUrl: https://api.screwdriver.cd
-  #   sonarHost: https://sonar.screwdriver.cd
-  #   adminToken: your-sonar-admin-token
-  #   sdUiUrl: https://cd.screwdriver.cd
-  #   sonarEnterprise: false
-  #   sonarGitAppName: "Screwdriver Sonar PR Checks"
+  default: "false"
+  plugin: sonar
+  sonar:
+    sdApiUrl: https://api.screwdriver.cd
+    sonarHost: https://sonar.screwdriver.cd
+    adminToken: your-sonar-admin-token
+    sdUiUrl: https://cd.screwdriver.cd
+    sonarEnterprise: false
+    sonarGitAppName: "Screwdriver Sonar PR Checks"
```

This is specific to our OSSD deployment. Should we make this as the default config for local development? Would it even work?
Should we leave this config commented out and just set `default: "false"`?

```suggestion
  sonar:
    sdApiUrl: https://api.screwdriver.cd
    sonarHost: https://sonar.screwdriver.cd
    adminToken: your-sonar-admin-token
    sdUiUrl: https://cd.screwdriver.cd
    sonarEnterprise: false
    sonarGitAppName: "Screwdriver Sonar PR Checks"
```